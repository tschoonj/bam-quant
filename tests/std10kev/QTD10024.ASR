File: C:\xrfdaten\2014KW30\chris\std10kev\std10024.spe
Start: 21.07.2014 15:57:12
Stop: 21.07.2014 15:58:52
BESSY: 298.485135 298.3901449
IONI: 1.451511E-10 1.447507E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 81653
Run: 100000
Dead: 18
ICR: 59950
OCR: 0
Accumulated BESSY: 59707.449603
Accumulated Ioni: 2.90502981E-08
N Accumulation: 200
Normfactor:       84315468.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cr
exp_x 112.020401000977 112.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  24   1      5.412   4187560.      2047.     127.37
