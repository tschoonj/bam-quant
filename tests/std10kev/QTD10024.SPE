$DATA:
0 2000
19136
12227
5385
1589
336
52
6
7
4
2
2
4
1
0
1
6
166
1163
1421
1294
1261
1156
1144
1192
1120
1096
1032
1030
944
977
947
943
954
907
907
820
821
810
794
797
799
768
727
731
725
727
753
790
678
699
683
745
675
653
764
678
711
669
666
704
709
639
705
630
670
627
661
668
671
686
620
626
609
641
638
662
600
592
599
582
606
607
574
571
552
598
595
538
581
577
604
575
606
619
603
648
612
641
654
641
677
712
656
685
653
679
631
626
595
617
589
675
627
630
610
600
607
550
591
598
601
572
625
548
580
596
588
650
726
696
645
665
617
625
628
598
585
602
578
597
610
662
716
863
992
1146
1291
1263
1131
932
796
702
663
640
625
684
676
670
710
721
656
739
651
639
695
660
638
616
643
689
650
667
725
700
866
970
1199
1548
2085
2776
3361
3846
3620
3170
2510
1788
1296
1027
843
762
719
710
723
714
729
745
742
753
733
756
801
838
829
881
939
1008
1102
1204
1230
1113
1120
1072
996
942
935
935
917
962
965
973
990
1055
1117
1052
1091
1127
1137
1228
1230
1205
1187
1248
1304
1258
1330
1327
1300
1311
1293
1247
1260
1379
1389
1401
1460
1557
1604
1734
1827
1872
2006
2180
2270
2547
2850
3226
3781
4674
6731
10607
19971
40177
78653
143874
238586
353252
467202
550161
573990
530234
434064
315492
203877
117161
60478
28406
12274
5581
2885
1972
1694
1758
1949
2193
2570
3409
4969
7768
12720
20835
32420
46571
62074
73654
79649
77471
66723
51549
36152
22662
12733
6438
3038
1331
571
251
120
90
99
101
109
168
222
318
463
545
679
593
626
510
364
247
148
97
69
53
39
30
25
20
24
17
23
18
21
11
25
28
21
21
19
23
25
26
31
67
88
92
104
108
95
103
72
53
43
36
31
13
21
11
13
9
16
10
20
23
24
24
23
21
21
26
30
19
19
15
15
12
13
11
11
12
12
17
10
12
10
10
11
11
14
10
11
19
17
19
19
33
15
21
24
17
11
10
10
13
12
11
14
17
19
31
18
12
20
19
13
15
15
17
12
19
17
19
22
17
15
24
22
15
21
18
13
14
21
17
21
18
10
15
12
18
21
25
25
21
23
29
29
32
46
34
31
32
18
11
17
9
12
21
18
13
15
15
19
18
14
22
26
20
19
28
31
37
26
33
48
40
48
39
68
57
69
88
115
139
211
215
289
347
416
529
573
620
645
642
602
534
469
387
320
201
149
109
64
62
36
21
24
19
6
16
15
22
11
18
22
26
26
29
26
31
38
49
57
100
158
248
413
547
877
1141
1464
1760
2071
2088
2135
1912
1686
1378
968
734
488
278
174
105
58
51
34
25
36
32
57
94
126
184
239
331
427
496
565
624
588
605
493
403
358
233
155
105
60
37
17
11
9
14
3
5
5
9
13
24
25
25
38
39
45
49
57
36
43
34
26
14
11
14
7
2
2
2
1
1
1
0
0
0
0
2
1
0
1
2
0
0
1
1
0
1
2
1
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
2
0
1
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
1
1
0
0
1
0
1
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
1
1
1
0
0
0
0
0
0
0
1
0
0
0
1
0
1
1
1
0
1
1
1
0
2
1
1
7
5
7
5
3
3
4
7
5
5
5
7
7
4
4
0
1
1
0
2
1
1
2
0
1
0
0
0
1
1
0
2
0
2
1
1
3
1
2
0
0
2
0
3
5
2
7
6
5
4
5
2
2
3
7
1
1
1
0
0
1
0
0
2
1
1
0
2
0
2
2
0
0
4
2
2
1
1
3
0
5
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
1
0
1
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10024.spe
Start: 21.07.2014 15:57:12
Stop: 21.07.2014 15:58:52
BESSY: 298.485135 298.3901449
IONI: 1.451511E-10 1.447507E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 81653
Run: 100000
Dead: 18
ICR: 59950
OCR: 0
Accumulated BESSY: 59707.449603
Accumulated Ioni: 2.90502981E-08
N Accumulation: 200
Normfactor:       84315468.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cr
 
exp_x 112.020401000977 112.0204
exp_y 150.265808105469 150.2658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  24   1      5.412   4187560.      2047.     127.37
