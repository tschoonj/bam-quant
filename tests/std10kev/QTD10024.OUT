     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:26:58  บ   
     บ  Spectrum: QTD10024.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        4 iterations done    บ   
     บ  ChiSquare =   330.6   last change =   .09%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -73.3 ๑   100.0   บ     -73.3 ๑      .9   บ   
     บ  GAIN (eV/ch)  บ     20.406 ๑   2.000   บ    20.406 ๑    .003   บ   
     บ det NOISE (eV) บ     -112.8 ๑    40.0   บ    -112.8 ๑    -3.0   บ   
     บ  FANO factor   บ       .285 ๑    .050   บ      .285 ๑    .004   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Cr-Ka                           4187560. ๑   2047.           บ   
     บ    KA1         5.412   1.00000     4187560. ๑   2047.    127.4  บ   
     บ              268.814    142.13        1095.           8.68E-01  บ   
     บ                                                                 บ   
     บ  2 Cr-Kb                            620663. ๑    789.           บ   
     บ    KB1         5.947   1.00000      620663. ๑    789.     82.6  บ   
     บ              295.032    153.15         728.           8.98E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
